isic/core/storages/utils.py

```diff
@@ -0,0 +1,5 @@
+from uuid import uuid4
+
+
+def generate_upload_to(instance, filename) -> str:
```

Comment:
For future investigation, I'm not sure why this is only used for superpixel masks and nowhere else.